pages/quizzes/html-quiz/html-quiz-result.html

```diff
@@ -0,0 +1,478 @@
+            </div>
+            <div
+              style="display: block"
+              class="btn btn-square-solid text-white bg-green-700 btn-bg-hover-green-600"
+            >
+              An empty tag doesn't require a closing tag
+            </div>
+            <div
+              style="display: block"
+              class="btn btn-square-outlined btn-bg-hover-cyan-600 text-hover-white text-cyan-600"
+            >
+              All of the above
+            </div>
+          </div>
+        </div>
+        <div class="spacer-3rem"></div>
+
+        <div class="bg-rose-200 text-rose-800 text-lg p-5">
+          <span><i class="fas fa-exclamation-triangle"></i> </span>
+          Oops, you are wrong. No point for this one.
+        </div>
+
+        <div class="spacer-3rem"></div>
+        <div class="spacer-3rem"></div>
+
+        <div class="flex justify-center align-items-center">
+          <h3 class="h3 inline-flex mr-5">Score</h3>
+          <h3 class="h3 font-bold inline-flex text-green-500">3</h3>
+        </div>
+      </div>
+    </main>
+    <!-- main -->
+
+    <!-- main -->
+    <!-- main section -->
+
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+
+    <!-- footer -->
+    <footer
+      class="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-cyan-200"
+      style="bottom: 0"
+    >
+      <div class="text-grey-700">
+        Made with
+        <span class="text-cyan-700 font-bold ml-1 mr-1">&lt;/&gt;</span> by
+
+        <a class="ml-1 text-hover-cyan-600">Ananya</a>
+      </div>
+      <div class="flex mt-3 align-items-center justify-center">
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-cyan-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
+          <i class="fab fa-linkedin mr-2 text-cyan-700"></i>
+        </a>
+      </div>
+
+      <div class="text-grey-700 mt-5">
+        Designed using
+
+        <a class="text-hover-cyan-600">Fusion CSS</a>
+      </div>
+    </footer>
+    <!-- footer -->
+
+    <script>
+      var modal = document.querySelector("#modal");
+
+      var modalBtn = document.querySelector("#modal-btn");
+
+      var closeBtn = document.querySelector(".modal-close-btn");
+      console.log(closeBtn);
+
+      modalBtn.onclick = function () {
+        modal.style.display = "block";
+      };
+
+      closeBtn.onclick = function () {
+        modal.style.display = "none";
+      };
+
+      window.onclick = function (event) {
+        if (event.target == modal) {
+          modal.style.display = "none";
+        }
+      };
```

Comment:
why are we adding JavaScript here in HTML file . It would be better if would could put this logic in a separate JS file.